http_cache/expiration.rst

```diff
@@ -85,5 +85,12 @@ the lifetime calculation vulnerable to clock skew. Another limitation
 of the ``Expires`` header is that the specification states that "HTTP/1.1
 servers should not send ``Expires`` dates more than one year in the future."
 
+.. note::
+
+    Accordingly with `RFC 7234 - Caching`_, the ``Expires`` header value is
+    ignored when ``s-max-age`` or ``max-age`` directives are defined.
```

Comment:
@xabbuh Minor comment, I don't get this change, it seems to me that both can be set at the same time, does it matter?